sklearn/cluster/dbscan_.py

```diff
@@ -96,6 +98,11 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
     <sklearn.neighbors.NearestNeighbors.radius_neighbors_graph>`
     with ``mode='distance'``.
 
+    Note about n_jobs: this does not seem to use multiple processors. Calls
+    NearestNeighbour, passing the n_jobs parameter to it, but NearestNeighbour
+    has not been parrallelized. Setting algorithm="brute" uses multiple
+    cores, but will cause a slow down.
```

Comment:
What do you think of :

Note about \`\`n_jobs\`\`: when \`\`algorithm="brute"\`\`, the \`\`n_jobs\`\` parameter is used to compute a brute force multiple processors approach which can cause a slow down. For a faster computation we recommend to use other NearestNeighbors algorithms by choosing another value for the parameter \`\`algorithm\`\`. Nevertheless, these  NearestNeighbors methods are not parrellelized in scikit-learn and will not use the \`\`n_jobs\`\` parameter.